manifests/v3.11.0/kube-apiserver/deployment.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: openshift-kube-apiserver
+  name: apiserver
+  labels:
+    app: openshift-kube-apiserver
+    apiserver: "true"
+spec:
+  strategy:
+    type: RollingUpdate
+  selector:
+    matchLabels:
+      app: openshift-kube-apiserver
+      apiserver: "true"
+  template:
+    metadata:
+      name: openshift-kube-apiserver
```

Comment:
should we add:

```
checkpointer.alpha.coreos.com/checkpoint: "true"
```

annotation here (it is used to tell the checkpointer to snapshot the pods)